spec/models/user_spec.rb

```diff
@@ -1,5 +1,89 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  before :all do
+    @user = User.create(full_name: 'Example Example', email: 'example@example.com', bio: 'example bo', image: 'image')
+    @user2 = User.create(full_name: 'ExampleNew Example', email: 'example2@example.com', bio: 'example bo', image: 'image')
```

Comment:
Use factory also: 
https://github.com/thoughtbot/factory_bot_rails
+ faker gem for unique fields:
https://github.com/faker-ruby/faker